### DISCUSSION TITLE
Regarding SNOPT/IPOPT during aviary installation

### DISCUSSION THREAD

**Discussion by h1he2li3** [OpenMDAO/Aviary#34] [General]
Hi, I was wondering if there is a way to Install Aviary without installing SNOPT, rather use IPOPT or a comparable open source package (any suggestions?)? Could you please help?

**Comment by johnjasa** [comment]
Hi there! Yes, you do not need SNOPT to run Aviary. You can use IPOPT installed with `pyOptSparse` and that is supported within Aviary. For some very simple problems, you can even use Scipy's SLSQP method.

**Reply by h1he2li3** [reply]
Hi john, Thank you. I will try to use use them and see how the installation goes.
